eng/scripts/Attest-CPEX-KPIs.ps1

```diff
@@ -0,0 +1,153 @@
+Set-StrictMode -Version 3
+. (Join-Path $PSScriptRoot "../common/scripts/Helpers/DevOps-WorkItem-Helpers.ps1")
+
+$Complete = 1
+$NA = 3
+
+$KPI_ID_Onboarding = "ba2c80d5-b8be-465f-8948-283229082fd1"
+$KPI_ID_Mgmt_Private_Preview = "e0504da9-8897-41db-a75f-5027298ba410"
+$KPI_ID_Data_Private_Preview = "dfe9c112-416e-4e0a-8012-4a3a29807782"
+$KPI_ID_Mgmt_Public_Preview = "84715402-4f3c-4dca-b330-f05206abaec5"
+$KPI_ID_Data_Public_Preview = "ad70777b-a1f5-4d77-8926-5c466d7a214d"
+$KPI_ID_Mgmt_GA = "210c095f-b3a2-4cf4-a899-eaab4c3ed958"
+$KPI_ID_Data_GA = "da768dff-8f90-4999-ad3a-adcd790911f3"
+
+function InvokeKustoCommand($command) {
+    $clusterUri = "https://azsdk-cpex-attestation.westus2.kusto.windows.net"
+    $databaseName = "CPEX_Attestation_DB"
+    $accessToken = az account get-access-token --resource "https://api.kusto.windows.net" --query "accessToken" --output tsv
+    $headers = @{ Authorization="Bearer $accessToken" }
+    $body = @{ csl = $command; db = $databaseName } | ConvertTo-Json -Depth 3
+    Invoke-RestMethod -Uri "$clusterUri/v1/rest/mgmt" -Headers $headers -Method Post -Body $body -ContentType "application/json"
+}
+
+function AddAttestationEntry($targetId, $actionItemId, $status, $targetType, $url) {
+    $tableName = "KpiEvidenceStream"
```

Comment:
AS discussed, let's change this to a param so we can create another test table and use it for testing.